yt/visualization/plot_window.py

```diff
@@ -982,7 +982,8 @@ def _setup_plots(self):
             # colorbar minorticks
             if f not in self._cbar_minorticks:
                 self._cbar_minorticks[f] = True
-            if self._cbar_minorticks[f] is True and MPL_VERSION < LooseVersion('2.0.0'):
+            if self._cbar_minorticks[f] is True and MPL_VERSION < LooseVersion('2.0.0') \
+                                                or self._field_transform[f] == symlog_transform:
```

Comment:
Good catch!

Super minor nit, can you put parentheses around this whole expression to avoid adding the continuation character?